src/actions/post-install.ts

```diff
@@ -0,0 +1,62 @@
+import picocolors from 'picocolors';
+import * as nodeFS from 'node:fs';
+import * as nodePath from 'node:path';
+import type { PackageManager } from './context.js';
+
+function getLintCommand(packageManager: PackageManager): string {
+	switch (packageManager) {
+		case 'npm':
+			return 'npm run stylelint';
+		case 'yarn':
+			return 'yarn stylelint';
+		case 'pnpm':
+			return 'pnpm stylelint';
+		case 'bun':
+			return 'bun run stylelint';
+		default:
+			return 'npm run stylelint';
+	}
+}
+
+function addStylelintScript(): void {
+	const packageJsonPath = nodePath.join(process.cwd(), 'package.json');
+
+	try {
+		const packageJson = JSON.parse(nodeFS.readFileSync(packageJsonPath, 'utf-8'));
+
+		if (!packageJson.scripts) {
+			packageJson.scripts = {};
+		}
+
+		if (!packageJson.scripts.stylelint) {
+			packageJson.scripts.stylelint = 'stylelint "**/*.css"';
```

Comment:
Since `stylelint` is automatically added to the `node_modules/.bin` folder you don't really have to a add new script named `stylelint` for [`${picocolors.green(lintCommand)}`](https://github.com/stylelint/create-stylelint/pull/92/files#diff-824834ecbbdc1bd87abac38b2389e83e86f2070a743de6ef57704a9417012e54R55).

i.e.
We should leave the naming of the script to the discretion of the user and update  `${picocolors.green(lintCommand)}` to be `${picocolors.green(lintCommand)} "**/*.css"`